flocx_market/tests/unit/test_matcher.py

```diff
@@ -0,0 +1,130 @@
+from unittest import TestCase
+from matcher import * 
+import json
+
+data = json.loads(""" {
```

Comment:
I would probably replace this giant chunk of data with a much smaller example dictionary. It should only contain what you need in order to run your tests.